app/services/confirm_enrolment.rb

```diff
@@ -1,13 +1,19 @@
 class ConfirmEnrolment
-  Response = Struct.new(:successful?, :application_choice)
+  attr_reader :errors, :application_choice
```

Comment:
I think we can 🔪 `:application_choice` here if we're hanging on to the reference  in the controller?